laboratorios/laboratorio-01/2019-1/pregunta-02/main.cpp

```diff
@@ -0,0 +1,195 @@
+#include <iostream>
+#include <iomanip>
+#include <cmath>
+#include <fstream>
+#include <cstring>
+
+void PopulateGameBoard(int t, char **tablero);
+void ShowGameBoard(int t, char **tablero);
+void SearchWord(char *word, int t, char **tablero, char *res);
+bool SearchRight(char *word, int t, char **tablero, char *res);
+bool SearchLeft(char *word, int t, char **tablero, char *res);
+bool SearchDown(char *word, int t, char **tablero, char *res);
+bool SearchUp(char *word, int t, char **tablero, char *res);
+
+
+int main() {
+    std::cout << "Ingresar el tamaño de la cuadricula: \n";
+    int t, cant;
+    std::cin >> t;
+    char **tablero, **word, res[3];
```

Comment:
`word` is an array of strings, so its name should be `words`.

Reply:
Can `res` be more descriptive? I think is a result, but not so sure what it contains.